rq_win/worker.py

```diff
@@ -28,7 +30,8 @@ def __init__(self, *args, **kwargs):
             kwargs['default_worker_ttl'] = 2
         super(WindowsWorker, self).__init__(*args, **kwargs)
 
-    def work(self, burst=False):
+    def work(self, burst=False, logging_level="INFO", date_format=DEFAULT_LOGGING_DATE_FORMAT,
+             log_format=DEFAULT_LOGGING_FORMAT):
```

Comment:
These new parameters should be passed through to the super implementation

Reply:
Something "upstream" was calling `WorkerClass.work` with a named argument of `logging_level` -- by not having this named parameter, things "fell over". I simply took the whole method signature to try and preserve as much compatibility as possible.

Reply:
Just pass the new parameters through to `super().work()` and this should be fine.

Reply:
Huh? This isn’t a call inside of ‘rq-win’ — something inside of ‘rq’ is calling the ‘work’ method (in the rq-win worker) with the named argument of ‘logging_level’.

Maybe I’m missing something here ...

Reply:
This `work()` method is overriding `Worker.work` implemented here: https://github.com/rq/rq/blob/master/rq/worker.py#L437

The reason you had to add the new parameters is because the base `work()` method now accepts them, and yes, something provided these parameters when it called `work()` on your WindowsWorker.

Just below here, on line 45, we call through to the base implementation of `work()` but we only provide the `burst` argument. We should provide the logging parameters as well:

```python
return super(WindowsWorker, self).work(burst=burst, logging_level=logging_level, date_format=date_format, log_format=log_format)
```

Reply:
Good catch! Thanks! Fixed!